plugins/plugin-github/che-plugin-github-factory-resolver/src/main/java/org/eclipse/che/plugin/github/factory/resolver/LegacyGithubURLParser.java

```diff
@@ -31,10 +31,6 @@ public LegacyGithubURLParser(URLChecker urlChecker) {
   @Override
   public GithubUrl parse(String url) {
     GithubUrl githubUrl = super.parse(url);
-    if (!urlChecker.exists(githubUrl.dockerFileLocation())) {
-      githubUrl.withDockerfileFilename(".codenvy.dockerfile");
-    }
-
     if (!urlChecker.exists(githubUrl.factoryJsonFileLocation())) {
       githubUrl.withFactoryFilename(".codenvy.json");
```

Comment:
Maybe its time to drop support of such? WSYT @skabashnyuk @benoitf

Reply:
Agree for codenvy files it was only something useful for avoid any breaks but now I would say this time is over